app/assets/stylesheets/card.scss

```diff
@@ -14,5 +14,6 @@ $card-bg-color: $sky-blue;
 }
 
 .selected {
-  background-color: $card-bg-color;
+  background-color: rgba(51, 15, 255, 0.7);
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.